pkgs/tools/misc/hdf4/default.nix

```diff
@@ -31,11 +31,12 @@ stdenv.mkDerivation rec {
     "-DHDF4_BUILD_WITH_INSTALL_NAME=OFF"
     "-DHDF4_ENABLE_JPEG_LIB_SUPPORT=ON"
     "-DHDF4_ENABLE_NETCDF=OFF"
-    "-DHDF4_ENABLE_SZIP_ENCODING=ON"
-    "-DHDF4_ENABLE_SZIP_SUPPORT=ON"
     "-DHDF4_ENABLE_Z_LIB_SUPPORT=ON"
     "-DHDF4_BUILD_FORTRAN=OFF"
     "-DJPEG_DIR=${libjpeg}"
+  ] ++ stdenv.lib.optionals (szip != null) [
+    "-DHDF4_ENABLE_SZIP_ENCODING=ON"
```

Comment:
@7c6f434c I think we already agreed that the maximum which Hydra can distribute is going to be distributed. Perhaps I shouldn't have expected more from this "first time contributor". 

I am OK with the current situation, although it would have been better if some things hadn't been said from my side.